mmv1/templates/terraform/examples/scc_v2_folder_mute_config_basic.tf.erb

```diff
@@ -0,0 +1,10 @@
+resource "google_pubsub_topic" "scc_v2_folder_mute_config" {
+  name = "<%= ctx[:vars]['topic_name'] %>"
+}
+
+resource "google_scc_v2_folder_mute_config" "<%= ctx[:primary_resource_id] %>" {
+  mute_config_id = "<%= ctx[:vars]['mute_config_id'] %>"
+  folder         = "<%= ctx[:test_env_vars]['folder_id'] %>"
```

Comment:
It looks like the resource requires this to be a real folder id. You should include the folder resource in the test config.

Reply:
addressed

Reply:
I still don't see a folder resource in the config.

Reply:
adressed